src/components/side_panel/find_and_replace/find_and_replace_store.ts

```diff
@@ -110,10 +111,11 @@ export class FindAndReplaceStore extends SpreadsheetStore implements HighlightPr
   }
 
   finalize() {
-    if (this.isSearchDirty) {
+    if (this.isSearchDirty && !this.isAutoSheetSwitch) {
```

Comment:
There is an issue: `this.activeSheetMatches` is no longer updated when switching to the next match on the next sheet :/

Reply:
@LucasLefevre done.